modules/apps/portal-background-task/portal-background-task-service/src/test/java/com/liferay/portal/background/task/internal/BaseBackgroundTaskTestCase.java

```diff
@@ -190,6 +190,22 @@ protected HashMap<String, Serializable> initializeThreadLocalValues() {
 		).build();
 	}
 
+	protected HashMap<String, Serializable> initializeThreadLocalValuesMixed() {
```

Comment:
@titokenzo I do not understand why you need to duplicate the logic in initializeThreadLocalValues

Reply:
If this is something only be used by specific test, then it should not be added to this base class